resources/lib/metadata.py

```diff
@@ -547,12 +547,13 @@ def get_year(api_data):
         return ''
 
     def get_mediatype(self, api_data, season=False):
-        """Get art dict from single item json api data"""
+        """Get mediatype from single item json api data"""
 
         # VRT NU Search API
         if api_data.get('type') == 'episode':
+            # Don't set mediatype on seasons to get a folder icon in the Kodi skin
             if season:
-                return 'season'
+                return ''
```

Comment:
How important is that folder icon? I think it would make sense to set the content type correctly on what it is, The way how it's rendered in the default skin is probably a design choice?

Reply:
This is not a remark, just a question. I think I'm also doing stuff like this in VTM GO.

Reply:
It's probably a bug in the Estuary-skin. The content type is set with `xbmcplugin.setContent()` I'm still figuring out what the purpose of the `mediatype` infoLabel is.

I think it's important that the user experience with the default skin is consistent.